notebooks/DrizzlePac/sky_matching/sky_matching.ipynb

```diff
@@ -4,14 +4,27 @@
    "cell_type": "markdown",
```

Comment:
<p>Hi <code>sky_matching_comparison.gif</code>  is missing. And the alt text revealing isn't very descriptive enough. Alt text is crucial for accessibility as it helps screen reader users understand the content of images/gif.</p><p> </p>

<pre class="ql-syntax" spellcheck="false">![Your descriptive text describing the gif](file)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>